packages/stylelint-plugin-mozaic/test/index.js

```diff
@@ -0,0 +1,72 @@
+var testRule = require('stylelint-test-rule-tape')
```

Comment:
make a test for : `media: '/@',`
and for all other cases ('kebab-case', lowerCamelCase, snake_case, PascalCase)